### ISSUE TITLE
Error when I testing a sortable component

### ISSUE THREAD

**Issue by Jucerweb** [clauderic/react-sortable-hoc#29]
When i running a test and this component has react-sortable-hoc inmediately throws this error  

````
react-sortable-hoc/dist/commonjs/utils.js:34
  return pre[0].toUpperCase() + pre.substr(1);
TypeError: Cannot read property '0' of undefined
````
I think this error is beacuse Jsdom, anyone has had this error ? any test works when the component has this lib

Help me :)

**Comment by framik76** [comment]
I have the same problem. I don't know what it could be. :(

**Comment by framik76** [comment]
I tryed to modify utils.js to check il variable 'pre' is undefined: 
return pre ? pre[0].toUpperCase() + pre.substr(1) : '';
and it works.
Can this fix to be merged, please?

**Comment by clauderic** [comment]
Fixed in `0.0.5`, just published to npm. Thanks
